.github/workflows/ci.yml

```diff
@@ -26,13 +26,13 @@ jobs:
         NXF_VER:
           - "23.04.0"
           - "latest-everything"
-        profile:
-          [
+        profile: [
             "test,docker --aligner alevin",
             "test,docker --aligner kallisto",
             "test,docker --aligner star",
-            "test,docker --aligner cellranger -stub",
-            "test,docker --aligner universc -stub",
+            "test,docker --aligner cellranger",
```

Comment:
What was the thing that was making it being - stub before?

Reply:
correct